diff --git a/node_modules/react-ace/lib/types.d.ts b/node_modules/react-ace/lib/types.d.ts
index 3fbd75b..569482c 100644
--- a/node_modules/react-ace/lib/types.d.ts
+++ b/node_modules/react-ace/lib/types.d.ts
@@ -123,7 +123,7 @@ export interface IAnnotation {
 }
 interface IRenderer extends Ace.VirtualRenderer {
     placeholderNode?: HTMLDivElement;
-    scroller?: HTMLDivElement;
+    scroller: HTMLDivElement;
 }
 export declare type IAceEditor = Ace.Editor & {
     renderer: IRenderer;
diff --git a/node_modules/react-ace/src/types.ts b/node_modules/react-ace/src/types.ts
index 08fe337..bc45000 100644
--- a/node_modules/react-ace/src/types.ts
+++ b/node_modules/react-ace/src/types.ts
@@ -130,7 +130,7 @@ export interface IAnnotation {

 interface IRenderer extends Ace.VirtualRenderer {
   placeholderNode?: HTMLDivElement;
-  scroller?: HTMLDivElement;
+  scroller: HTMLDivElement;
 }

 export type IAceEditor = Ace.Editor & {